examples/logger.py

```diff
@@ -85,11 +108,19 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame = ConfirmClearingFrame()
         self.confirmFrame.confirmed.connect(self.clearLog)
+        # define handler
+        self.handler = LoggingHandler(self.addLog)
+        # arbitrary format
+        fs ="%(name)s %(message)s"
+        formatter = logging.Formatter(fs)
+        self.handler.setFormatter(formatter)
+        logging.getLogger().addHandler(self.handler)
```

Comment:
Is the logger in `qiwis.py` being called in well?

It seems not to print the logs generated in `qiwis`.
Also I think an appropriate name is necessary in `getLogger()`.

Reply:
In order to print log message, the logger level should be set as logging.INFO in logger.py, because logger.info is used in qiwis.py to propagate log message.

From @kangz12345 I told to test root logger, so I just used root logger. But in order to manage logger level like INFO, DEBUG, it seems that name needed.

 I will set the name as 'parent' temporarily, and change logger level to show you message is well printed in next update.

Reply:
Oh, you're right! I didn't know `logging.getLogger()` would return the root logger.
Thanks for your detailed description😃